Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorHelpers.h

```diff
@@ -92,6 +92,10 @@ namespace AzToolsFramework
         //! to the current Editor Focus Mode and Container Entity setup.
         bool IsSelectableInViewport(AZ::EntityId entityId) const;
 
+        //! Returns whether the entityCacheId can be selected in the viewport according
+        //! to the current Editor Focus Mode and Container Entity setup.
+        bool IsSelectableInViewport(size_t entityCacheId) const;
```

Comment:
update: I wouldn't make this a public function and instead just make a wrapper in the .cpp, and then call it from line 333 with `entityCacheIndex` directly). If it does make it easier to have this be a member function then private would be better.

Reply:
I just removed the function completely since it was only called in the entityId version either way,

Reply:
See my later comment which is why I think it is worth keeping (albeit private) as it'll be faster to call in the loop as we already have the `entityCacheIndex`